testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jca/statistics/DataSourceJdbcStatisticsTestCase.java

```diff
@@ -81,20 +81,6 @@ public void doSetup(final ManagementClient managementClient) throws Exception {
             }
         }
 
-        @Override
-        public void tearDown(final ManagementClient managementClient, final String containerId) throws Exception {
-            try {
-                StringBuffer sb = assertStatisticsSet(true, getStatAddr(dsAddress)).append(
```

Comment:
With your change, there is no longer these assertions when the test is tear down. They should not have been there in the first place but maybe they should be moved to the end of the test instead?

Reply:
I don't think they are particularly important assertions, it is basically just a test that basic management functionality works